### DISCUSSION TITLE
Collapse left navigation sidebar

### DISCUSSION THREAD

**Discussion by hakonhagland** [sphinx-doc/sphinx#13029] [Q&A]
Is there any sphinx theme that supports collapsing the left navigation sidebar bar by pressing a collapse/expand button? Like this https://docs.openedx.org/en/latest/educators/how-tos/sidebar_collapse_expand.html

A  screenshot from that page:

![image](https://github.com/user-attachments/assets/87dd9d94-e4ac-48ed-9e79-45b0db0bba8f)

This can be useful if the reader wants to focus on the text and not be distracted by the navigation bar taking up a large part of the screen. I know this exists on mobile devices, but I am interested in larger screens also (like my laptop).

**Comment by mgeier** [comment]
There are multiple themes with collapsible side bar. You can click through the examples at https://sphinx-themes.org/ to check out the exact behavior.

Note that some themes have both left and right sidebar and sometimes only one of them is collapsible.

Note also that the behavior on small screens is different in some themes.
